.github/workflows/composite-actions/setup-kind-cluster/action.yaml

```
@@ -21,6 +21,10 @@ inputs:
   istio-version:
     required: true
     description: The version of Istio
+  k8sgateway_api_version:
```

```suggestion
  k8sgateway-api-version:
```

The existing inputs follow this convention. I can't find the github docs now, but I had tried to base it off them. If I was wrong and your convention is preferred, can we change the other inputs in this file so we're consistent?

---

Updated in the appropriate places. Left the underscores in the env files as that is the usual formatting for env variables, and the switch in conventions comes when the env variable is assigned to the action input.